.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
```

Comment:
Please rename the file too

Reply:
Done: https://github.com/elementor/elementor-icons/pull/70/commits/c761ec21850fea81bd705be1ec11f1085fcf263b